### ISSUE TITLE
Error in validGP(gp) : must specify only one of 'font' and 'fontface'

### ISSUE THREAD

**Issue by stefvanbuuren** [growthcharts/chartplotter#7]
After updating R packages to R 4.2.1 I get

``` r
library(chartplotter)
fn <- system.file("extdata", "bds_v1.0", "smocc", "Laura_S.json", package = "jamesdemodata")
ind <- bdsreader::read_bds(fn)

g <- process_chart(ind, chartcode = "NJAA",
                   dnr = "smocc", period = c(0.5, 1.1667), nmatch = 10, break_ties = TRUE)
#> New names:
#> • `NA.` -> `NA....30`
#> • `NA..1` -> `NA..1...31`
#> • `NA..2` -> `NA..2...32`
#> • `NA..3` -> `NA..3...33`
#> • `NA.` -> `NA....47`
#> • `NA..1` -> `NA..1...48`
#> • `NA..2` -> `NA..2...49`
#> • `NA..3` -> `NA..3...50`
#> • `NA..4` -> `NA..4...51`
#> • `NA.` -> `NA....65`
#> • `NA..1` -> `NA..1...66`
#> • `NA..2` -> `NA..2...67`
#> • `NA..3` -> `NA..3...68`
#> • `NA..4` -> `NA..4...69`
grid::grid.draw(g)
#> Error in validGP(gp): must specify only one of 'font' and 'fontface'
```

<sup>Created on 2022-08-16 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

The output "New names:"  should not be there. Also, the `validGP` error is novel.

**Comment by stefvanbuuren** [comment]
The `validGP` was caused by a few `font =` statements in the package that designs the growth charts. That one is solved now.

**Comment by stefvanbuuren** [comment]
The `New names:` note was caused by different behaviour of the `predict()` function in the `brokenstick 2.3.0` package. The bug gave rise to incorrect predictions.

Solved in `chartplotter 0.30.0`.
